Tests/Controller/ControllerTest.php

```diff
@@ -325,4 +325,22 @@ public function testGetTranslationsFromCompilerPassAnotherBundle()
 JSON
             , $response->getContent());
     }
+
+    public function testGetTranslationsWithThreeLettersLocale()
+    {
+        $client  = static::createClient();
+
+        $crawler  = $client->request('GET', '/translations/messages.json?locales=ach_UG');
+        $response = $client->getResponse();
+
+        $this->assertEquals(<<<JSON
+{
+    "fallback": "en",
+    "defaultDomain": "messages",
+    "translations": {"ach_UG":{"messages":{"hello":"hello"}}}
```

Comment:
@bdunogier travis breaks here. 

As there is no ach_UG translation file messages should be empty : "translations": {"ach_UG":[]}